examples/demo/main.rs

```diff
@@ -185,6 +188,25 @@ impl<'a> App<'a> {
     }
```

Comment:
You must add `self.image_scale_state` above here too. Cycling the images with `o` in the demo helps spot all sort of edge case bugs.